### ISSUE TITLE
High CPU load when the plugin is active

### ISSUE THREAD

**Issue by ihulsbus** [AndreyVMarkelov/jira-prometheus-exporter#36]
Hi, 

We are seeing relatively high CPU load on our testing instance of JIRA when activating the plugin. 
![screenshot 2018-11-16 at 11 22 40 am](https://user-images.githubusercontent.com/11390889/48618605-33514680-e99a-11e8-95a9-21f8b6cc62e5.png)
around 10:35 the plugin was installed, and we observed a significant rise in CPU load. To check if this was due to the plugin, we disabled it around 11:08, to enable it again at 11:18. we did the same between 11:24 and 11:30. The logs did not show any errors, and the exporter further worked as expected, exposing metrics. 

has there been any reports of this behaviour before? 

Cheers,

**Comment by AndreyVMarkelov** [comment]
Not. But please setup zero minutes in settings to check attachments. To see
what will be?

пт, 16 нояб. 2018 г., 14:29 Ian Hulsbus notifications@github.com:

**Comment by ihulsbus** [comment]
Sorry, can you elaborate on what you want me to do? 
Thanks

**Comment by AndreyVMarkelov** [comment]
@ihulsbus Try please with the latest version

**Comment by ihulsbus** [comment]
Sorry for my late reply @AndreyVMarkelov. I will try and test as soon as I can get my hands on a staging instance.

**Comment by AndreyVMarkelov** [comment]
@ihulsbus any update?

**Comment by ihulsbus** [comment]
Hi Andrey, 

Sorry for the delay. I went on holiday before I got the chance of testing the plugin. 
In the mean time, my colleague has tested the plugin on our staging instance, and the high CPU load seems to be resolved there. 

We will activate the plugin in production soon, and if that gives similar results as currently in our staging instance we should be good to go :) 

The screenshot below is the result of our staging instance with the plugin activated in the morning,
plugin stopped at 12:28 and started again at 15:21. 
<img width="1494" alt="Screenshot 2019-07-16 at 16 44 03" src="https://user-images.githubusercontent.com/11390889/62532417-9b0b3280-b844-11e9-8aa2-5926d6c06fd3.png">
